src/fetch/rewards.py

```diff
@@ -0,0 +1,170 @@
+"""Functionality for computing solver rewards
+
+The only function used in other parts of the code is `compute_solver_rewards` for generating
+aggregate reward information per solver."""
+
+from pandas import DataFrame, Series
+from src.fetch.prices import TokenConversion
+
+EPSILON_UPPER = 12000000000000000
+EPSILON_LOWER = 10000000000000000
+PERIOD_BUDGET_COW = 250000 * 10**18
+CONSISTENCY_REWARD_CAP_ETH = 6 * 10**18
+QUOTE_REWARD_COW = 6 * 10**18
+QUOTE_REWARD_CAP_ETH = 6 * 10**14
```

Comment:
6 COW and 0.0001 ETH? Seems like a very low number.

Reply:
This is the reward _per quote_ and it is indeed 6 COW or 0.0006 ETH.